### PR TITLE
CLDR-17417 v45 BRS: algorithmic locales

### PR DESCRIPTION
**Pull request by srl295**
CLDR-17417

I normally run CLDRModify `-fp` but that is disabled per #3502 - anything else to do here?

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: common/annotations/sr_Latn.xml [hunk 1]
```diff
@@ -1426,7 +1426,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="🧑‍🍼" type="tts">osoba hrani bebu</annotation>
 		<annotation cp="👼">anđeo | bajka | beba | lice | mašta</annotation>
 		<annotation cp="👼" type="tts">beba anđeo</annotation>
-		<annotation cp="🎅">bajka | božić | božić bata | Deda Mraz | proslava</annotation>
+		<annotation cp="🎅">bajka | božić | božić bata | deda mraz | Deda Mraz | proslava</annotation>
```
Comment (macchiati) [inline common/annotations/sr_Latn.xml:1429]: I think you also want to run -fP, which should remove the case duplicates here.

**Review by macchiati — CHANGES_REQUESTED** [review]

File: common/annotations/sr_Latn.xml (hunk 1 above)
Comment (srl295) [reply common/annotations/sr_Latn.xml:1429]: `-fP` did not remove the case duplicates here.

Comment (macchiati) [reply common/annotations/sr_Latn.xml:1429]: Hmmm. I'll file a ticket for that.

**Review by macchiati — APPROVED** [review]

File: common/annotations/sr_Latn.xml (hunk 1 above)
Comment (pedberg-icu) [reply common/annotations/sr_Latn.xml:1429]: FYI, a later CLDRModify pass did remove the case dup here

**Pull request merged**